app/assets/javascripts/work_items.es6

```diff
@@ -0,0 +1,18 @@
+export default class WorkItems {
+  constructor() {
+    this.toggle_show_items()
+  }
+
+  toggle_show_items(){
+    let button = $("#work-items-button")
+    let show = I18n.t('hyrax.works.form.show_child_items')
```

Comment:
Should there also be a translation by default in `en.hyrax` for this?  When I desk check I get `missing "en.hyrax.works.form.hide_child_items" translation`

Reply:
Ooops. Missed committing locale file.

Reply:
Works locally now.